pkg/iris/scan.go

```diff
@@ -212,3 +220,16 @@ func (s *scanner) ShareStatus(configid string, workspaceId string, accessToken s
 	}
 	return nil
 }
+
+func (s *scanner) ShareAdminLogs(configid string, workspaceId string, accessToken string, loglevel string, message string) error {
+	var statusArray []StatusItem
+	statusObject := NewAdminLogEvent(configid, s.runId, workspaceId, loglevel, message)
+	statusArray = append(statusArray, *statusObject)
+	statusByte, err := storage.Marshal(statusArray)
+	err = s.irisApi.PostStatus(statusByte, accessToken)
```

Comment:
Same endpoint, same eventhub instance for status and admin logs, right?

Reply:
Yes. The endpoint is the same. No change from IRIS side.